### ISSUE TITLE
 login details not showing [Environment Sandbox] when testing

### ISSUE THREAD

**Issue by burton22ful** [AlexDisler/cordova-plugin-inapppurchase#224]
I am testing with ios. When calling   .restorePurchases() it asks for login details.  Should it also say [Environment Sandbox] also because is it not saying that?  I am using a development build.

Please include:

### Your code / steps to reproduce
```js

```

**Comment by shochberg** [comment]
I don't think it would say [Environment Sandbox] until after you have entered your login credentials successfully.  Apple knows it's a sandbox login by the login that you use.
